src/main/java/jenkins/branch/UntrustedBranchProperty.java

```diff
@@ -120,6 +121,7 @@ public UntrustedBranchProperty(String[] publisherWhitelist) {
     /**
      * Our {@link Descriptor}.
      */
+    @Symbol("untrustedCode")
```

Comment:
I also don't see this show up as an option for Multibranch Pipelines, so IDK if we want a symbol or not. In this case, I think it's because Pipelines extend `Job` rather than `Project`, but I'm not sure.

If we do keep the symbol, maybe `untrustedBranches` would be better?

Reply:
I'll remove it if it's not configurable. Probably best to keep it consistent with UI

Reply:
It might be configurable for some other implementation of MultiBranchProject, but I'm not sure what those might be.

Reply:
Cool, I'll double check before I do anything